heppy/Request.py

```
@@ -42,7 +47,7 @@ def add_subtags(self, parent, tags):
         return parent
 
     @staticmethod
-    def build(data):
+    def build(data) -> Request:
```

Ruff (0.11.9)

Undefined name `Request`

(F821)